src/components/AnacondaWizard.jsx

```diff
@@ -373,78 +383,68 @@ const Footer = ({
         }
     };
 
-    const goToPreviousStep = (activeStep, onBack, errorHandler) => {
+    const goToPreviousStep = () => {
         // first reset validation state to default
-        setIsFormValid(true);
+        setIsFormValid(false);
```

Comment:
how is this change related?

Reply:
anyhow that's correct change - but it looks like unrelated to this.

Reply:
Yes, its unrelated. I remove it, but i take a look at this and isStepFollowedBy after merge.